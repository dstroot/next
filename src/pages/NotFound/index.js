import React, { useEffect } from 'react';
// component
import SectionNotFound from '../../components/Sections/SectionHero/SectionNotFound';
import Footer from '../../components/Footer';

const NotFound = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `NEXT â€¢ Not Found`;
    window.scrollTo(0, 0);
  });

  return (
    <div>
      <SectionNotFound />
      <Footer />
    </div>
  );
};

export default NotFound;
