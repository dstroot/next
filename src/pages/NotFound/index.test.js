import React from 'react';
import { create } from 'react-test-renderer';
import NotFound from '../NotFound';

describe('NotFound', () => {
  it('it should render', () => {
    let tree = create(<NotFound />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
