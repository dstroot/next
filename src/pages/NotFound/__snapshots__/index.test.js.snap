// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound it should render 1`] = `
<div>
  <div
    className="background"
    style={
      Object {
        "backgroundImage": "url(notfound.svg)",
      }
    }
  >
    <div
      className="divText"
    >
      <div
        className="container"
      >
        <div
          className="row"
        >
          <div
            className="col text-center"
          >
            <h1
              className="nxt_heading-xlarge"
            >
              Sorry, we couldn’t find that page!
            </h1>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-8 offset-md-2 text-center"
          >
            <p
              className="mt-4 circular"
            >
              It’s ok to get lost sometimes. Let’s get you back on track.
            </p>
            <a
              aria-current="page"
              href="/"
              onClick={[Function]}
            >
              <button
                className="btn btn-primary nxt_rounded-pill nxt_body-small px-4 py-2"
              >
                Go to Homepage
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container-fluid bg-dark"
  >
    <div
      className="row"
    >
      <div
        className="col p-3"
      >
        <img
          alt="Logo"
          className="pr-3 pb-4"
          src="NXT_logo_footer_dark.svg"
        />
        <span
          className="nxt_footer"
        >
          <p>
            By using this website you accept our
             
            <a
              href="/terms"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Terms of Use
              </span>
            </a>
            ,
             
            <a
              href="/social"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Social Media Guidelines
              </span>
            </a>
            , and
             
            <a
              href="/privacy"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Privacy Policy
              </span>
            </a>
          </p>
        </span>
        <small>
          <div
            className="nxt_footer"
          >
            <p>
              No bank guarantee • Not a deposit • May lose value • Not FDIC/NCUA insured • Not insured by any federal government agency
            </p>
            <p>
              Pacific Life, its affiliates, its distributors, and respective representatives do not provide tax, accounting, or legal advice. Any taxpayer should seek advice based on the taxpayer's particular circumstances from an independent tax advisor or attorney.
            </p>
            <p>
              <i>
                Pacific Life is a product provider. It is not a fiduciary and therefore does not give advice or make recommendations regarding insurance or investment products.
              </i>
            </p>
            <p>
              Pacific Life Insurance Company (Newport Beach, CA) is licensed to issue insurance products in all states except New York. Product availability and features may vary by state. Fixed annuity products are available through licensed third parties.
            </p>
            <p>
              © 
              2019
               Pacific Life Insurance Company, Newport Beach, CA.
            </p>
          </div>
        </small>
      </div>
    </div>
  </div>
</div>
`;
